contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -3,15 +3,11 @@ pragma solidity 0.8.26;
 
 import { EnumerableSet } from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
 import { PausableUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/PausableUpgradeable.sol";
-// solhint-disable-next-line max-line-length
-import { ReentrancyGuardUpgradeable } from "@openzeppelin/contracts-upgradeable-v4/security/ReentrancyGuardUpgradeable.sol";
-// solhint-disable-next-line max-line-length
-import { ERC20SnapshotUpgradeable } from "@openzeppelin/contracts-upgradeable-v4/token/ERC20/extensions/ERC20SnapshotUpgradeable.sol";
```

Comment:
Think we no longer use @openzeppelin/contracts-upgradeable-v4/ - would double check but if so - it could be removed from dependencies and yarn.lock

Reply:
Yes, removed from yarn.lock. 
and the PR has already removed it from dependencies in package.json.